pkg/scheduler/extender/extender_test.go

```diff
@@ -905,6 +909,149 @@ func Test_createReservation(t *testing.T) {
 	assert.Equal(t, len(capacityRequests), len(reservationResource.Spec.ReservationRequests))
 }
 
+func Test_calculate(t *testing.T) {
+	events := []coreV1.Event{
```

Comment:
please also change the Reason with the const define in this test code.  it should always be sync with the code it tested.

Reply:
sounds Good, use EventReason variables in test code.